lib/patches/abstract_mysql_adapter.rb

```diff
@@ -0,0 +1,10 @@
+# TODO:
+# MySQL 5.7+ does not allow NULL primary keys, but older versions of Rails
+# create primary key columns DEFAULT NULL.
+#
+# Remove this file and its environment.rb reference after upgrading to Rails 4.
+#
+# https://github.com/rails/rails/pull/13247#issuecomment-32425844
+class ActiveRecord::ConnectionAdapters::AbstractMysqlAdapter
```

Comment:
I went with this class since it's specifically overriding this: https://github.com/yahonda/rails/blob/v3.2.19/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L114